CHANGELOG.md

```diff
@@ -3,13 +3,46 @@
 
 ## Changelog
 
-
 All notable changes to the graphistry-helm repo are documented in this file. Additional Graphistry components are tracked in the main [Graphistry major release history documentation](https://graphistry.zendesk.com/hc/en-us/articles/360033184174-Enterprise-Release-List-Downloads).
 
 The changelog format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and all PyGraphistry-specific breaking changes are explictly noted here.
 
 ## [Development]
+*   Added Dask operator to control our Dask cuda Scheduler and Workers. 
+    This will allow us to scale up and down the number of workers as needed. 
+    Temporary workaround for service name issue with the operator, 
+    currently unable to set the service name to `dask-scheduler` in the scheduler.service spec 
+    so we are using a service named `dask-scheduler` instead. 
```

Comment:
Is the same name right? perhaps there is some different name here (_dask-scheduler_)

Reply:
this is from last versions part of changelog

Reply:
lemme remove